package.json

```diff
@@ -19,7 +19,7 @@
   },
   "gitHead": "ef12b4cdd358fd5dfcab16a3328dfacf9e636027",
   "dependencies": {
-    "utils": "github:wilsonpage/utils",
+    "utils": "github:financial-times/ft-app-utils",
     "event": "github:wilsonpage/event",
```

Comment:
D'oh I've just realised there is 2 event libraries (event vs evt)!

Reply:
Yes! And we've not forked `mixin` either but because they're not mentioned on this - https://financialtimes.atlassian.net/browse/AT-4368?focusedCommentId=482368 maybe we need to ask Wilson again?